package.json

```diff
@@ -12,25 +12,31 @@
     "test": "jest",
```

Comment:
the diff is showing formatting changes, is pre-commit configured?

Reply:
It wasn't. I'm going to activate it and reformat this file.